### PR TITLE
Split `AccountsPage/Accounts/Accounts.module.css`.

### PR DESCRIPTION
**Pull request by amassarwi**
This splits `AccountsPage/Accounts/Accounts.module.css` into smaller css modules.

### CODE REVIEW

File: app/components/views/AccountsPage/Accounts/AccountRow/AccountDetails/AccountDetails.jsx [hunk 1]
```diff
@@ -2,21 +2,15 @@ import { FormattedMessage as T } from "react-intl";
...
 import { IMPORTED_ACCOUNT, DEFAULT_ACCOUNT } from "constants";
+import DataLine from "./DataLine";
+import styles from "./AccountDetails.module.css";
 
 const isHidable = (account) =>
   account.accountName !== IMPORTED_ACCOUNT &&
   account.accountName !== DEFAULT_ACCOUNT &&
   !account.total;
 
-const DataLine = ({ children }) => (
```
Comment (amassarwi) [inline app/components/views/AccountsPage/Accounts/AccountRow/AccountDetails/AccountDetails.jsx:13]: moved to a separate component.

File: app/components/views/AccountsPage/Accounts/AccountRow/Header/Header.jsx [hunk 2]
```diff
@@ -54,40 +50,4 @@ const Header = React.memo(
   }
 );
 
-const Row = ({
```
Comment (amassarwi) [inline app/components/views/AccountsPage/Accounts/AccountRow/Header/Header.jsx:57]: moved to `AccountRowContent`

File: app/components/views/AccountsPage/Accounts/AccountsList/AccountsList.jsx [hunk 3]
```diff
@@ -1,27 +1,9 @@
-import { FormattedMessage as T } from "react-intl";
-import AccountRow from "./AccountRow";
+import AccountRow from "../AccountRow";
 import { DecredLoading } from "indicators";
-import { InfoDocModalButton } from "buttons";
 import { Subtitle } from "shared";
-import style from "./Accounts.module.css";
-
-const SubtitleInfoIcon = React.memo(() => (
```
Comment (amassarwi) [inline app/components/views/AccountsPage/Accounts/AccountsList/AccountsList.jsx:8]: moved to a separate component - `SubtitleInfoIcon`

File: app/components/views/AccountsPage/Accounts/AccountsList/AccountsList.jsx [hunk 4]
```diff
@@ -1,27 +1,9 @@
...
-    <InfoDocModalButton
-      document="BalanceOverviewInfo"
-      modalClassName={style.infoFields}
-      double
-      draggable
-    />
-  </div>
-));
-
-const SubtitleWalletName = React.memo(({ walletName }) => (
```
Comment (amassarwi) [inline app/components/views/AccountsPage/Accounts/AccountsList/AccountsList.jsx:19]: moved to `SubtitleWalletName`

**Review by alexlyp — APPROVED** [review]

**Pull request merged**